docker/moonbeam.Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Node for Moonbase Alphanet.
```

Comment:
Why is there a second dockerfile now?

Reply:
This is the public docker image file.
The other one (the previous one) is for internal use only

Reply:
Also I need to change the desc here